lib/rspec/support/ruby_features.rb

```diff
@@ -65,6 +65,12 @@ def supports_exception_cause?
         end
       end
 
+      def ripper_supported?
+        # JRuby supports Ripper but it seems to be unstable
+        # and also reports wrong line number on JRuby 9.0.0.0.
+        defined?(RUBY_ENGINE) && RUBY_ENGINE == 'ruby' && RUBY_VERSION >= '1.9.2'
```

Comment:
> Would it be possible to write some logic that detects if ripper is working on JRuby so that we allow it to be used if a user is running a future version of JRuby where the ripper issues have been fixed? Although, the obvious ways to do that would involve loading ripper and testing it out, which I don't think we want to do...

I added examples to check if the Ripper behavior in rspec-support's spec rather than user's environement so that we can immediately notice the change on Travis CI in the future. By the way I noticed that we do not have CI for JRuby 9.0.0.0.